plugins/qtAccessionPlugin/lib/model/QubitAccession.php

```diff
@@ -81,6 +81,12 @@ public function isAccrual()
     return 0 < count(QubitRelation::get($criteria));
   }
 
+  public static function maskEnabled()
+  {
+    $setting = QubitSetting::getByName('accession_mask_enabled');
+    return (null === $setting || boolval($setting->getValue(array('sourceCulture' => true))));
```

Comment:
This feels wrong to me, are we sure that if the setting is missing (a rare case) we should consider that the mask is enabled? Otherwise, we could use sfConfig to get its value.

Reply:
This setting doesn't get created by a migration, alas, and the default behaviour IIRC is to use the mask.

Reply:
Ah, nice! Thanks